cmake/modules/FindPyArrow.cmake

```diff
@@ -24,7 +24,7 @@ execute_process(
           "from __future__ import print_function\ntry: import pyarrow; print(' '.join(pyarrow.get_libraries()), end='')\nexcept:pass"
   OUTPUT_VARIABLE __pyarrow_libraries)
 
-# And the version
+  # And the version
 execute_process(
   COMMAND "${Python_EXECUTABLE}" -c
```

Comment:
we don't really need to do any version matching, we should just link all against arrow_python.dylib or arroy_python.so since arrow will include both a numbered and unnumbered version and we want the version picked up to be dictated by python dependency checking anyway.

Reply:
Arrow 1.0.1 does not include both such versions, as above.